CONTRIBUTORS.md

```diff
@@ -1,17 +1,16 @@
 <!--
                   
 #
-# This source file is part of the TemplatePackage open source project
+# This source file is part of the SpeziOneSec open source project
 #
-# SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+# SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
 #
 # SPDX-License-Identifier: MIT
 # 
              
 -->
 
-TemplatePackage contributors
+SpeziOneSec contributors
 ====================
 
-* [Paul Schmiedmayer](https://github.com/PSchmiedmayer)
-* [Vishnu Ravi](https://github.com/vishnuravi)
+* [Lukas Kollmer](https://github.com/lukaskollmer)
```

Comment:
Let's add @LambdaDigamma + @PSchmiedmayer here as well 🚀

Reply:
oops sorry for removing you 😬